packages/core/test/integration/query-interface/schemas.test.ts

```diff
@@ -179,13 +179,16 @@ describe('QueryInterface#{create,drop,list}Schema', () => {
     expect(postDeletionSchemas).to.not.include(testSchema, 'dropSchema did not drop testSchema');
   });
 
+  // For Oracle Database, users are considered as schema. listSchemas() doesn't allow to fetch the
+  // defaultSchema.
```

Comment:
Is there not a way to get this? Possibly by manually including the default schema?